### ISSUE TITLE
Python 3 Rewrite

### ISSUE THREAD

**Issue by cpressland** [dcramer/mangodb#59]
Python 2 is EoL soon. Pls rewrite to the 3's.

**Comment by phw** [comment]
I don't think that is the right approach. Python 3 is already over 10 years old, this is really not fitting for mangodb, which is on the top edge of technological development. Rather I'd say go directly with Python 4.

**Comment by cpressland** [comment]
Why not Python 6 at least? ![](https://preview.redd.it/ipvod31laab01.jpg?width=960&crop=smart&auto=webp&s=1601eea44f40c38f048a3dd2c0bcc73978d3733d)
